[INFO   ] Logger: Record log in C:\Users\rumeza\.kivy\logs\kivy_22-02-07_5.txt
[INFO   ] deps: Successfully imported "kivy_deps.angle" 0.3.1
[INFO   ] deps: Successfully imported "kivy_deps.glew" 0.3.0
[INFO   ] deps: Successfully imported "kivy_deps.sdl2" 0.3.1
[INFO   ] Kivy: v2.0.0
[INFO   ] Kivy: Installed at "C:\Users\rumeza\AppData\Local\Programs\Python\Python39\lib\site-packages\kivy\__init__.py"
[INFO   ] Python: v3.9.4 (tags/v3.9.4:1f2e308, Apr  6 2021, 13:40:21) [MSC v.1928 64 bit (AMD64)]
[INFO   ] Python: Interpreter at "C:\Users\rumeza\AppData\Local\Programs\Python\Python39\python.exe"
[INFO   ] Logger: Purge log fired. Analysing...
[INFO   ] Logger: Purge 8 log files
[INFO   ] Logger: Purge finished!
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "C:\Users\rumeza\PycharmProjects\pythonProject\main.py", line 2, in <module>
[WARNING] stderr:     import kivy.garden.matplotlib.backend_kivyagg as file
[WARNING] stderr:   File "C:\Users\rumeza\AppData\Local\Programs\Python\Python39\lib\site-packages\kivy\garden\__init__.py", line 180, in load_module
[WARNING] stderr:     return self._load_module(fullname, moddir)
[WARNING] stderr:   File "C:\Users\rumeza\AppData\Local\Programs\Python\Python39\lib\site-packages\kivy\garden\__init__.py", line 183, in _load_module
[WARNING] stderr:     mod = imp.load_module(fullname, None, moddir,
[WARNING] stderr:   File "C:\Users\rumeza\AppData\Local\Programs\Python\Python39\lib\imp.py", line 244, in load_module
[WARNING] stderr:     return load_package(name, filename)
[WARNING] stderr:   File "C:\Users\rumeza\AppData\Local\Programs\Python\Python39\lib\imp.py", line 216, in load_package
[WARNING] stderr:     return _load(spec)
[WARNING] stderr:   File "C:\Users\rumeza\.kivy\garden\garden.matplotlib\__init__.py", line 1, in <module>
[WARNING] stderr:     from kivy.garden.matplotlib.backend_kivy import FigureCanvasKivy,\
[WARNING] stderr:   File "C:\Users\rumeza\.kivy\garden\garden.matplotlib\backend_kivy.py", line 256, in <module>
[WARNING] stderr:     from matplotlib import _png
[WARNING] stderr: ImportError: cannot import name '_png' from 'matplotlib' (C:\Users\rumeza\AppData\Local\Programs\Python\Python39\lib\site-packages\matplotlib\__init__.py)
